src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -114,46 +124,52 @@ public function handle(AddProductToOrderCommand $command)
 
         $this->addProductToCart($cart, $product, $combination, $command->getProductQuantity());
 
+        // Fetch Cart Product
+        $productCart = $this->getCartProductData($cart, $product, $command->getProductQuantity());
+
         $invoice = $this->createNewOrEditExistingInvoice(
```

Comment:
I maybe dont get it. We have 1 invoice per product ?

Reply:
No it's depending of the input ;)